client/src/components/AddNewTaskBtn/style.module.css

```diff
@@ -0,0 +1,11 @@
+.button {
```

Comment:
This might be a nit, but I think we should use a more specific name here unless these attributes are gonna be applied to all the buttons around.

Reply:
Ok, thank you for the feedback. I will update this class.